wordpress-mobile/WordPress-iOS
# Readability Part 2, life without WPTableViewCell

Part II of readability support actually starts to follow readable margins and content guides.

For the most part, we're really only tackling views and controllers that can support readability without many changes. We have cleaned up a few spots though, with a bit of use of `UIStackView`.

For this PR, note:

1. First requires a review, merge and updating to WP-iOS-Shared pod 0.6.1 via PR: https://github.com/wordpress-mobile/WordPress-Shared-iOS/pull/104
2. Also requires first merging and then pulling the changes from WP-iOS https://github.com/wordpress-mobile/WordPress-iOS/pull/5756 (Those commits are included here as well and can be diffed out via updating or just ignored for review)

To test:

(1.) Ensure the following views/controllers, on iPad, are following the default readable margins in both landscape and portrait orientation:

- Blog list
- Blog Details
  - Plans
  - Comments
  - Sharing
  - Settings
- Me tab
  - My Profile
  - Account Settings
  - App Settings
  - Notification Settings
  - Help & Support
- Reader tab
  - Followed Sites "Manage"
- Post Settings
  - Categories (with parent/child indentation)
  - Featured image
  - Location

(2.) Ensure the following views/controller are using the original margin hack (until they can be adapted):

- Notifications
- CommentViewController (when viewing a single comment)
- Pages
- Blog Posts
- People

Note: Reader posts/cards are untouched for this go around.

Needs review: @frosty can I push your eyes to the limit for another review? 😁 👀
